ci/gpu/build.sh

```diff
@@ -167,13 +167,6 @@ if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
             echo "Running GoogleTest $test_name"
             ${gt} --gtest_output=xml:"$WORKSPACE/test-results/"
         done
-
```

Comment:
This file is no longer in use by CI since the switch to GitHub Actions.

the new CPP test file is here: https://github.com/rapidsai/cudf/blob/branch-23.02/ci/test_cpp.sh.

I imagine you may want to make similar changes there.

We will be removing all of the old Jenkins CI scripts from the RAPIDS repositories in the coming weeks.

Reply:
thanks, done.